.projenrc.ts

```diff
@@ -27,7 +27,7 @@ import {
 const GITHUB_USER = 'awslabs';
 const PUBLICATION_NAMESPACE = 'cdklabs';
 const PROJECT_NAME = 'generative-ai-cdk-constructs';
-const CDK_VERSION: string = '2.88.0';
+const CDK_VERSION: string = '2.118.0';
```

Comment:
```suggestion
const CDK_VERSION: string = '2.114.0';
```

Reply:
done in [e8ebbf5](https://github.com/awslabs/generative-ai-cdk-constructs/pull/180/commits/e8ebbf58d89f9699465624722f45c3eb9401228f)